CHANGELOG.md

```diff
@@ -50,6 +50,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
  - #1783, Include partitioned tables into the schema cache. Allows embedding, UPSERT, INSERT with Location response, OPTIONS request and OpenAPI support for partitioned tables - @laurenceisla
  - #1878, Add Retry-After hint header when in recovery mode - @gautam1168
+ - #1917, Add error codes with the `"PGRST"` prefix to the error response body to differentiate PostgREST errors from PostgreSQL errors and include the `detail` and `hint` fields with `null` values if they are missing from the error response body. - @laurenceisla
```

Comment:
This should be split into two CHANGELOG entries for more clarity.

```suggestion
 - #1917, Add error codes with the `"PGRST"` prefix to the error response body to differentiate PostgREST errors from PostgreSQL errors - @laurenceisla
 - #1917, Normalize the error messages by always having the `detail` and `hint` error fields as `null - @laurenceisla
```

Reply:
I modified it slightly to clarify when they will have the null value.